Engines/Wine/Verbs/DXVK/script.js

```diff
@@ -39,45 +36,49 @@ class DXVK {
 
         print("NOTE: wine version should be greater or equal to 3.10");
 
-        if (operatingSystemFetcher.fetchCurrentOperationSystem() != "Linux") {
-            wizard.message(
-                tr(
-                    "DXVK might not work correctly on macOS. This is depending on Metal api support and MoltenVK compatibility layer advancement"
-                )
-            );
-        } else {
-            wizard.message(
-                tr(
-                    "Please ensure you have the latest drivers (418.30 minimum for NVIDIA and mesa 19 for AMD) or else DXVK might not work correctly."
-                )
-            );
+        if (operatingSystemFetcher.fetchCurrentOperationSystem().getFullName() !== "Linux")
+        {
+            const question = tr("DXVK is currently unsupported on non-Linux operating systems due to MoltenVK implementation being incomplete. Select how do you want to approach this situation.")
+            const choices = [
+                tr("YES, continue with DXVK installation regardless"),
+                tr("NO, quit script alltogether"),
+                tr("Exit DXVK Installer, but continue with the script")
+            ];
+
+            const answer = wizard.menu(question, choices);
+
+            switch (answer.index) {
+                case 1:
+                    // choice: "NO, quit script alltogether"
+                    throw new Error("User aborted the script.");
+                case 2:
+                    // choice: "Exit DXVK Installer, but continue with the script"
+                    return this;
+                default:
+                    // do nothing
+            }
         }
+        else {
+            wizard.message(tr("Please ensure you have the latest drivers (418.30 minimum for NVIDIA and mesa 19 for AMD) or else DXVK might not work correctly."));
+        }
+        const githubDownloader = new GitHubReleaseDownloader("doitsujin", "dxvk")
```

Comment:
This must be:
```Javascript
const githubDownloader = new GitHubReleaseDownloader("doitsujin", "dxvk", wizard);
```
The `.withWizard` is outdated.

Reply:
Understood